public/javascripts/control.js

```diff
@@ -92,6 +92,10 @@
         {
             $headline.children('.controlLabel').text(properties.kind.value);
         }
+        else if (type == 'audit')
+        {
+            $headline.children('.controlLabel').text("Audit");
+        }
```

Comment:
Audit controls get a fixed label "Audit". Users can choose a separate data name (default: "audit" instead of "untitled").